schema.go

```diff
@@ -1,8 +1,72 @@
 package kallax
 
+import "fmt"
+
+// Schema represents a table schema in the database. Contains some information
+// like the table name, its columns, its identifier and so on.
 type Schema interface {
-	Alias() string
-	Table() string
-	Identifier() string
-	Columns() []string
+	// GetAlias returns the name of the alias used in queries for this schema.
+	GetAlias() string
```

Comment:
why we added here `Get` prefix? if you have a implementation of this interface with the name `Alias` make it private, but don't add this ugly `Get` prefix